src/samples/ProofOfEntitlement/ProofOfEntitlement.tsx

```diff
@@ -15,120 +15,179 @@ import Button from '../../components/BaseComponents/Button/Button';
 declare const PCore;
 declare const myLoadMashup: any;
 
-export default function ProofOfEntitlement(){
+export default function ProofOfEntitlement() {
+  const [entitlementData, setEntitlementData] = useState(null);
+  const [showNoAward, setShowNoAward] = useState(false);
+  const [showProblemWithService, setShowProblemWithService] = useState(false);
+  const history = useHistory();
+  const { t } = useTranslation();
+  const currency = new Intl.NumberFormat('en-GB', { style: 'currency', currency: 'GBP' });
```

Comment:
This code is already concise and standard way to format, can we make it more reusable by exposing in our utils?

Reply:
yes that's a great idea, we can do that. since it was only required in this scenario I just added it to POE only